my-app/src/app/main-layout/content-projection/course/course.component.ts

```diff
@@ -1,22 +1,22 @@
 import { DatePipe } from '@angular/common';
-import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { Course } from 'src/app/models/course';
 
 @Component({
   selector: 'app-course',
   templateUrl: './course.component.html',
   styleUrls: ['./course.component.scss']
 })
-export class CourseComponent implements OnInit {
+export class CourseComponent {
 
-  @Input() course: Course;
+  // Task 2
+  @Output() private deleteCourse = new EventEmitter<number>();
```

Comment:
as far as Output is a some kind of external api, it should be public, as far as you use it in parent component

Reply:
to delete course it's better to delete by id, it's a string usually, it is not obligatory to fix now, but take a look at this point

Reply:
did not quite get it. I think I am emitting an id only. are you suggesting to change the type of id to string?